airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -224,6 +224,49 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+  SelectiveAuthenticator:
+    title: Selective Authenticator
+    description: Authenticator that selects concrete authenticator based on config property.
+    type: object
+    additionalProperties: true
```

Comment:
is there a reason this can't be false?

Reply:
From I saw most of places has it true. In general in my experience of working with pydantic, additionalProperties=true helps to keep backward compatibility. The only place were it false in declarative_component_schema is root level.
WDYT?

Reply:
fair enough!